kotlin/workflow-host/src/main/java/com/squareup/workflow/internal/WorkflowNode.kt

```diff
@@ -38,7 +38,7 @@ import kotlin.coroutines.experimental.CoroutineContext
  * [Workflow.initialState].
  */
 internal class WorkflowNode<InputT : Any, StateT : Any, OutputT : Any, RenderingT : Any>(
```

Comment:
If `WorkflowContext.compose` no longer knows about `StateT`, how is `WorkflowNode` learning about it? Where does it come from?

Reply:
It basically doesn't, it's effectively `Any`. But it doesn't need to in practice. The generic param is basically just there so the `Workflow` methods all agree.

Reply:
Let me see if I understand: the `StateT` of the root `WorkflowNode` comes from `WorkflowHost.Factory#run$workflow_host`. But of every other `WorkflowNode` in the tree has `StateT` of `Any`? Given that, can you push this PR farther and get rid of `StateT` from `WorkflowNode` and `RealWorkflowContext`?